Dockerfile

```diff
@@ -60,6 +60,7 @@ COPY supervisor /etc/supervisor/
 COPY --from=builder --chown=ids:www-data /home/app/appbuild /home/ids/appbuild
 RUN mv /home/ids/appbuild /home/ids/${APP_NAME} && \
     cd /home/ids/${APP_NAME} && pip3 install -r /root/requirements.txt && \
+    apt-get remove -y gcc && \
```

Comment:
Per discussion with Tony in this [Slack convo](https://harvard-huit.slack.com/archives/CP2FA1N5R/p1739980167604749), we should be using a method similar to the one in the [comments](https://at-harvard.atlassian.net/browse/ETD-484) to remove undesired libraries.  Tony suggested that this way causes the image to be bloated.

Reply:
Also what about curl?  Is it needed in the container?  It should probably be removed if not.